src/simulator_core/entities/assets/demand_cluster.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self.temperature_return: float = DEFAULT_TEMPERATURE - DEFAULT_TEMPERATURE_DIFFERENCE
         self.temperature_return_target = self.temperature_return
         self.pressure_input = DEFAULT_PRESSURE
-        self.thermal_power_allocation = 0
+        self.thermal_power_allocation = 50000.0
```

Comment:
Why this default value? Shouldn't we then use a DEFAULT_POWER?

Reply:
Aangepast

Reply:
We kunnen nog steeds een discussie hebben over de zin/onzin van deze default properties.